### DISCUSSION TITLE
Task List - Mark tasks referencing pull requests as done when the PR is merged

### DISCUSSION THREAD

**Discussion by t6d** [community/community#4562] [Projects and Issues]
It'd be great if pull requests would be treated like issues in task lists. That is, I'd like tasks referencing a PR to be marked as done when the PR is merged.

**Comment by matschaffer** [comment]
I'd love something similar to this where I could have a task list check box get checked when all issues referenced in the task are closed.

**Comment by stemis** [comment]
This would be very useful in PR's as well

We have a lot of PR's that can only be merged when the underlying PR is merged as well. Marking an item as done in the task list once the PR is merged makes this a lot easier, for example:

`- [ ] Merge #1`

In addition to this, it may be useful to have an option to automatically convert a PR from draft to ready when all task items are ticked off.
